src/objects.rs

```diff
@@ -27,10 +27,8 @@ pub enum ObjectShape {
 /// Also see the [TMX docs](https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#tmx-object).
 #[derive(Debug, PartialEq, Clone)]
 pub struct ObjectData {
-    /// ID of the object, which is unique per map since Tiled 0.11.
-    ///
-    /// On older versions this value is defaulted to 0.
-    pub id: u32,
+    id: u32,
+    // TODO: make public, differ somewhat from get_tile
```

Comment:
Why not make it public in this change? Is it more involved?

Reply:
Whether it is public or not depends on if we should check for ID uniqueness and fail if there are duplicates.

Reply:
I think it's fine to have `id` private, but this comment is about `tile`.

Reply:
Oh. Yeah, forgot to make it public. The issue is the name, should rename it to tile_data or something.